tests/testAPI.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+#
+#*******************************************************************************
+#* Copyright (c) 2019, 2020 IBM Corporation and others.
+#*
+#* Licensed under the Apache License, Version 2.0 (the "License");
+#* you may not use this file except in compliance with the License.
+#* You may obtain a copy of the License at
+#*
+#*    http://www.apache.org/licenses/LICENSE-2.0
+#*
+#* Unless required by applicable law or agreed to in writing, software
+#* distributed under the License is distributed on an "AS IS" BASIS,
+#* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#* See the License for the specific language governing permissions and
+#* limitations under the License.
+#*******************************************************************************/
+#
+
+KRUIZE_URL="http://localhost"
+KRUIZE_PORT="31313"
+AUTH_TOKEN=""
+
+#Check if jq exists
+if ! [ -x "$(command -v jq)" ]; then
+  echo "Error: jq is not installed."
+  exit 1
+fi
+
+#Check if Kruize is running without issues, and fetch recommendations generated
+while [[ $(curl -s -H "Authorization:Bearer $AUTH_TOKEN" "$KRUIZE_URL:$KRUIZE_PORT/health") == "UP" ]]; do
```

Comment:
Suggest separating this out into functions, eg: one function that returns a list of all applications being monitored by Kruize, another that returns the info for a app that is passed as an arg, another that checks the validity of the AUTH TOKEN etc.